[2025-03-11T09:29:21.375+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-11T09:29:21.439+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tiki_extract_dag.crawl_dien-tu-dien-lanh manual__2025-03-11T08:53:49.647616+00:00 [queued]>
[2025-03-11T09:29:21.484+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tiki_extract_dag.crawl_dien-tu-dien-lanh manual__2025-03-11T08:53:49.647616+00:00 [queued]>
[2025-03-11T09:29:21.485+0000] {taskinstance.py:2867} INFO - Starting attempt 4 of 4
[2025-03-11T09:29:21.559+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): crawl_dien-tu-dien-lanh> on 2025-03-11 08:53:49.647616+00:00
[2025-03-11T09:29:21.582+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=508) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-11T09:29:21.588+0000] {standard_task_runner.py:72} INFO - Started process 529 to run task
[2025-03-11T09:29:21.630+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'tiki_extract_dag', 'crawl_dien-tu-dien-lanh', 'manual__2025-03-11T08:53:49.647616+00:00', '--job-id', '572', '--raw', '--subdir', 'DAGS_FOLDER/tiki_extract_dag.py', '--cfg-path', '/tmp/tmpk3rfgard']
[2025-03-11T09:29:21.633+0000] {standard_task_runner.py:105} INFO - Job 572: Subtask crawl_dien-tu-dien-lanh
[2025-03-11T09:29:21.890+0000] {task_command.py:467} INFO - Running <TaskInstance: tiki_extract_dag.crawl_dien-tu-dien-lanh manual__2025-03-11T08:53:49.647616+00:00 [running]> on host 5c8190d7da62
[2025-03-11T09:29:52.885+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tiki_extract_dag' AIRFLOW_CTX_TASK_ID='crawl_dien-tu-dien-lanh' AIRFLOW_CTX_EXECUTION_DATE='2025-03-11T08:53:49.647616+00:00' AIRFLOW_CTX_TRY_NUMBER='4' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-11T08:53:49.647616+00:00'
[2025-03-11T09:29:52.905+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-11T09:29:52.906+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-11T09:29:52.908+0000] {logging_mixin.py:190} INFO - Current task name:crawl_dien-tu-dien-lanh state:running start_date:2025-03-11 09:29:21.448450+00:00
[2025-03-11T09:29:52.914+0000] {logging_mixin.py:190} INFO - Dag name:tiki_extract_dag and current dag run status:running
[2025-03-11T09:29:52.915+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-11T09:29:53.767+0000] {job.py:229} INFO - Heartbeat recovered after 32.66 seconds
[2025-03-11T09:29:56.749+0000] {tiki_extract_dag.py:139} INFO - Starting crawl for category dien-tu-dien-lanh (ID: 4221)
[2025-03-11T09:29:56.780+0000] {tiki_extract_dag.py:141} INFO - Calling API: https://tiki.vn/api/v2/products with params: {'page': 1, 'limit': 40, 'category': '4221'}
[2025-03-11T09:29:57.353+0000] {tiki_extract_dag.py:150} INFO - Total pages for dien-tu-dien-lanh: 50 (limited to 50)
[2025-03-11T09:30:27.290+0000] {connectionpool.py:868} WARNING - Retrying (Retry(total=4, connect=5, read=4, redirect=None, status=None)) after connection broken by 'ReadTimeoutError("HTTPSConnectionPool(host='tiki.vn', port=443): Read timed out. (read timeout=15)")': /api/v2/products/167558939
[2025-03-11T09:30:40.215+0000] {job.py:229} INFO - Heartbeat recovered after 35.59 seconds
[2025-03-11T09:30:40.251+0000] {local_task_job_runner.py:266} INFO - Task exited with return code -9. For more information, see https://***.apache.org/docs/apache-***/stable/troubleshooting.html#LocalTaskJob-killed
[2025-03-11T09:30:40.376+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-11T09:30:40.384+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
